scripts/aad/aad-app-reg.sh

```diff
@@ -504,14 +504,14 @@ JSON
 
   redirectUris="\"http://localhost:8000/api/docs/oauth2-redirect\""
 
-  if [[ -n ${replyUrl} ]]; then
-      echo "Adding reply/redirect URL \"${replyUrl}\" to ${appName} Swagger UI app"
-      redirectUris="${redirectUris}, \"${replyUrl}\""
+  if [[ -n ${treUrl} ]]; then
+      echo "Adding reply/redirect URL \"${treUrl}\" to ${appName} Swagger UI app"
+      redirectUris="${redirectUris}, \"${treUrl}\", \"${treUrl}/api/docs/oauth2-redirect\""
   fi
 
   swaggerUIApp=$(jq -c . << JSON
 {
-    "displayName": "${appName} Swagger UI",
```

Comment:
Right at the bottom of this file we output some ENV vars, 1 called SWAGGER_UI_CLIENT_ID. This no longer makes sense. Can you add a story to tidy that up if you are not changing it in this PR?